src/BuildScriptGeneratorCli/readme.md

```diff
@@ -0,0 +1,18 @@
+# Oryx Cli Structure
+## Background
+Due to the transition to net7, we found that the past cli package isn't compatible with the new framework. Hence, we've decided to move to `System.CommandLine` package to develop our cli. For developers, the structure of the cli codes are different then the past.
```

Comment:
nit: for `.md` files, I always like to cap the line length at 120 characters to make it easier to read in GitHub reviews :smile: